templates/runner.yaml

```diff
@@ -0,0 +1,146 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Actually, it may be the case that this instance isn't connected to gitlab forever, for example if we moved to a different service, so maybe we should leave the naming agnostic about CI?

Reply:
I did change this template name to `gitlab-runner.yaml` after all